package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "peggy",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

I think, that should be changed on release
```suggestion
  "version": "1.1.0",
```

---

I disagree, for reasons written elsewhere.  I don't want to belabor you by repeating, but I'm going to leave this open so that it can be found if we decide against the standard approach.

---

I retreat from what I said here.

I'm beginning to realize that there are two issues:

1. Should the package version be updated in this PR?  (no)
2. Should the package version be updated in source, from the package, in general?  (yes)

In my prior reply I was responding to 2, but I now believe you meant 1

In the case of 1 I agree and will do as requested

Sorry for the misunderstanding

---

On the understanding that 1 was incorrect, 1.2.0 is now rolled back to 1.1.0

If the point was about the automation, I'd like to continue the discussion

---

>     2\. Should the package version be updated in source, from the package, in general?  (yes)

Agreed with that, just to clarify, that change should always be done manually (or by the release preparation script), not on each CI run